src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx

```diff
@@ -57,7 +65,11 @@ Your Python application code also needs to be monitored by the [Python APM agent
 
 Click the CodeLens text to see related charts.
 
-![A screenshot of an expanded code-level metrics CodeLens that shows related charts.](./images/code-level-charts.png "An expanded code-level metrics CodeLens that shows related charts.")
+<img
+  title="An expanded code-level metrics CodeLens that shows related charts."
```

Comment:
Not sure if we capitalize these, but I just wanted to point it out in case we don't!